src/js/accordion/translations.js

```diff
@@ -0,0 +1,14 @@
+const Translations = {
+  open_all: {
+    fi: 'Avaa kaikki',
```

Comment:
Why is this twice in the code, once here and once in .po files?

Reply:
Prevent title from flashing with bad translation